projects/1.16/assets/rankine/lang/zh_cn.json

```
@@ -0,0 +1,2526 @@
+  "entity.rankine.beaver": "河狸",
+  "item.rankine.solar_flare_spawn_egg": "Solar Flare刷怪蛋",
+  "entity.rankine.solar_flare": "Solar Flare",
+  "item.rankine.shrouded_king_spawn_egg": "Shrouded King刷怪蛋",
+  "entity.rankine.shrouded_king": "Shrouded King",
+
+  "entity.minecraft.villager.rankine.metallurgist": "冶金学家",
+  "entity.minecraft.villager.rankine.mineralogist": "矿物学家",
+  "entity.minecraft.villager.rankine.botanist": "植物学家",
+  "entity.minecraft.villager.rankine.gem_cutter": "Gem Cutter",
+
+  "enchantment.rankine.puncture": "刺穿",
+  "enchantment.rankine.puncture.desc": "直接攻击造成额外的穿甲伤害",
+  "enchantment.rankine.blast": "爆裂",
+  "enchantment.rankine.blast.desc": "使锤子锤击时能使一定范围内的方块爆裂",
+  "enchantment.rankine.atomize": "粉碎",
+  "enchantment.rankine.atomize.desc": "在锤碎方块时，若第一次尝试产生产物时没有产品，则再次尝试产生产物",
+  "enchantment.rankine.lightning_aspect": "雷电附加",
+  "enchantment.rankine.lightning_aspect.desc": "在攻击目标时引发闪电",
+  "enchantment.rankine.swing": "挥锤",
+  "enchantment.rankine.swing.desc": "减少15％的锤击冷却时间(附加的).",
+  "enchantment.rankine.excavate": "范围挖掘",
+  "enchantment.rankine.excavate.desc": "增加每次锤击破坏的方块数量",
+  "enchantment.rankine.daze": "击晕",
+  "enchantment.rankine.daze.desc": "使目标获得缓慢效果，完美攻击会增强效果",
+  "enchantment.rankine.dune_walker": "沙丘行者",
+  "enchantment.rankine.dune_walker.desc": "增加在沙子上的移动速度",
+  "enchantment.rankine.snow_drifter": "雪地行者",
+  "enchantment.rankine.snow_drifter.desc": "增加在雪上的移动速度",
+  "enchantment.rankine.speed_skater": "速滑运动员",
+  "enchantment.rankine.speed_skater.desc": "增加在冰上的移动速度",
+  "enchantment.rankine.gas_protection": "气体保护",
+  "enchantment.rankine.gas_protection.desc": "减少气体的有害影响",
+  "enchantment.rankine.quake": "地震",
+  "enchantment.rankine.quake.desc": "根据深度增加挖掘速度",
+  "enchantment.rankine.foraging": "觅食",
+  "enchantment.rankine.foraging.desc": "增加挖土时获得各种物品的机会",
+  "enchantment.rankine.impact": "冲击(长矛)",
+  "enchantment.rankine.impact.desc": "增加长矛投掷时的击退",
+  "enchantment.rankine.antiquated": "古董",
+  "enchantment.rankine.antiquated.desc": "在主手或副手时增加幸运属性",
+  "enchantment.rankine.cleanse": "净化",
+  "enchantment.rankine.cleanse.desc": "根据目标所带有的药水效果数量造成额外伤害",
+  "enchantment.rankine.endpoint": "传送",
+  "enchantment.rankine.endpoint.desc": "将玩家传送至长矛落点，高等级附魔能减少传送时的伤害",
+  "enchantment.rankine.withering_curse": "凋零诅咒",
+  "enchantment.rankine.withering_curse.desc": "受到伤害(除魔法伤害和凋零伤害)会导致你获得短时间的凋零效果",
+  "enchantment.rankine.ghast_regeneration": "恢复",
+  "enchantment.rankine.ghast_regeneration.desc": "恢复药水可以恢复耐久",
+  "enchantment.rankine.prying": "除你武器",
+  "enchantment.rankine.prying.desc": "有几率使击中的目标手上拿的东西掉落，概率取决于物品的稀有度",
+
+  "jei.description.crushing_basic": "可以用机器或青铜锤破碎矿石得到",
+  "rankine.jei.crushing": "破碎",
+  "rankine.jei.alloy": "合金化",
+  "rankine.jei.triple_alloy": "高级合金化",
+  "rankine.jei.sluicing": "淘金",
+  "rankine.jei.evaporation": "蒸发",
+  "rankine.jei.beeoven": "蜂窝式炉",
+  "rankine.jei.element": "元素细节",
+  "rankine.jei.finery": "精炼铁",
+  "itemGroup.rankine_world": "Project Rankine建筑",
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

请使用标准格式。
```suggestion
  "itemGroup.rankine_world": "Project Rankine｜建筑",
```

---

模组名能翻译就翻译呗，兰金计划/项目什么的，结合模组内容